sqlglot/dialects/bigquery.py

```diff
@@ -202,10 +202,17 @@ def _unix_to_time_sql(self: BigQuery.Generator, expression: exp.UnixToTime) -> s
 def _build_time(args: t.List) -> exp.Func:
     if len(args) == 1:
         return exp.TsOrDsToTime(this=args[0])
-    if len(args) == 3:
-        return exp.TimeFromParts.from_arg_list(args)
+    elif len(args) == 2:
```

Comment:
```suggestion
    if len(args) == 2:
```

Reply:
Curious what's the reasoning here?

Reply:
`elif` is redundant because of the `return`, this pattern is pretty common in python

Reply:
yep let's switch for style consistency